### PR TITLE
More menu transitions 

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-4008
Continuing transition. Few more items 

image
image
image
image

### CODE REVIEW

File: src/components/profile/AvatarCircle.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default function AvatarCircle({
     ios || !isAvatarPickerAvailable ? React.Fragment : ContextMenuButton;
 
   return (
-    <Wrapper handlePressMenuItem={handlePressMenuItem} menuConfig={items}>
+    <Wrapper menuConfig={items} onPressMenuItem={handlePressMenuItem}>
```
Comment (tchayen) [inline src/components/profile/AvatarCircle.js:102]: What's the difference `handlePress...` vs `onPress...`?

**Review by tchayen — APPROVED** [review]
I think it looks good on both OS-es.

File: src/components/profile/AvatarCircle.js (hunk 1 above)
Comment (osdnk) [reply src/components/profile/AvatarCircle.js:102]: I just renamed this for consistency

**Review by perunt — APPROVED** [review]

**Review by terrysahaidak — APPROVED** [review]

**Pull request merged**